examples/btc-example/README.md

```diff
@@ -0,0 +1,71 @@
+# How to use TatumSDK with BTC
+
+These examples should guide you through some basic operations of the BTC blockchain. You will be able to:
+
+- generate BTC wallet
+- send BTC to another wallet
+- check the balance of your wallet
+
+- generate virtual account for BTC
+- assign deposit address to virtual account
```

Comment:
not assign, but generate - we are generating, not assigning - its different operation